src/MaRRTPathPlanNode.py

```diff
@@ -803,7 +810,7 @@ def getFrontConeObstacles(self, map, frontDist):
         frontDistSq = frontDist ** 2
 
         frontConeList = []
-        for cone in map.cones:
+        for cone in map:
```

Comment:
based on my other comments here will be:
`for cone in track.cones`
note, additional renamings are needed